internal/sirius/edit_payment_test.go

```diff
@@ -0,0 +1,105 @@
+package sirius
+
+import (
+	"fmt"
+	"github.com/pact-foundation/pact-go/dsl"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+)
+
+func TestEditPayment(t *testing.T) {
+	t.Parallel()
+
+	pact := newPact()
+	defer pact.Teardown()
+
+	testCases := []struct {
+		name          string
+		setup         func()
+		cookies       []*http.Cookie
+		expectedError func(int) error
+	}{
+		{
+			name: "OK",
+			setup: func() {
+				pact.
+					AddInteraction().
+					Given("I have an lpa which has been paid for").
+					UponReceiving("A request to edit a payment").
+					WithRequest(dsl.Request{
+						Method: http.MethodPut,
+						Path:   dsl.String("/lpa-api/v1/payments/123"),
+						Body: map[string]interface{}{
+							"id":          dsl.Like(123),
```

Comment:
Whilst it doesn't matter if the ID is sent in the request, I don't think we should include it in the contract since it isn't (shouldn't be) used by the API.